presto-geospatial/src/main/java/com/facebook/presto/plugin/geospatial/GeoFunctions.java

```diff
@@ -243,8 +243,10 @@ public static double stLength(@SqlType(GEOMETRY_TYPE_NAME) Slice input)
     @SqlType(DOUBLE)
```

Comment:
- please squash the commits 
- fix commit message. `Optimize Min Max geospatial functions` is too generic, instead please describe what you changed which resulted with better performance of min max geospatial functions.

Reply:
Also put benchmark results into the commit message.

Reply:
Thank you. I fixed the commit message itself saying which functions are being optimized. I have also included the benchmark numbers in the comment.

I will try to figure out how I can squash the commits tomorrow.